tests/test_config_type_def.py

```diff
@@ -259,3 +292,19 @@ def __init__(self, a: typing.Union[submodule, ClassConfig[submodule]],
                         {'a': {'a': 1, 'b': 2}, 'b': {'a': 3, 'b': 4, 'c': 5}}).b.c == 5
     assert TypeDef.load(ClassConfig[module],
                         {'a': {'a': 1, 'b': 2}, 'b': {'a': 3, 'b': 4, 'c': 5}}).build().b._c == 5
+
+
+test_path()
```

Comment:
I just removed this because I found it causes import issues to do so in case some test failed.